contracts/interfaces/ISwitchboard.sol

```diff
@@ -51,4 +51,12 @@ interface ISwitchboard {
     function getMinFees(
         uint32 dstChainSlug
     ) external view returns (uint256 switchboardFee, uint256 verificationFee);
+
+    function setFees(
+        uint256 nonce_,
+        uint32 dstChainSlug_,
+        uint256 verificationFees_,
+        uint256 switchboardFees_,
+        bytes calldata signature_
+    ) external;
```

Comment:
These functions dont fall under the ISwitchboard interface. Interface is what socket needs to work with them. setFees is an internal function and depends on how sb is built. Same applies to transmit manager and execution manager interfaces.